manifests/charts/cloudcore/values.yaml

```diff
@@ -126,7 +126,7 @@ controllerManager:
   resources:
     limits:
       cpu: 200m
-      memory: 50Mi
+      memory: 100Mi
     requests:
       cpu: 100m
-      memory: 25Mi
+      memory: 50Mi
```

Comment:
if modify this, better also fix values in https://github.com/kubeedge/kubeedge/blob/master/manifests/profiles/version.yaml
these two files need keeping the same with each other generally